pkg/doc/did/doc.go

```diff
@@ -747,7 +747,7 @@ type signatureSuite interface {
 	GetDigest(doc []byte) []byte
 
 	// Verify will verify signature against public key
-	Verify(pubKey *verifier.PublicKey, doc []byte, signature []byte) error
+	Verify(pubKey interface{}, doc []byte, signature []byte) error
```

Comment:
this change *only* makes sense if pubKey is an opaque object to every package except for a kms implementation.

Reply:
and I also see resolve actually constructing a generic value... (so pubkey doesn't seem to be opaque at all).